src/telemetry/AriaTelemetry.ts

```diff
@@ -805,7 +806,7 @@ class AriaTelemetry {
             Event: '',
             ExceptionDetails: '',
             ElapsedTimeInMilliseconds: '',
-            ACSChatVersion: require('@azure/communication-chat/package.json').version // eslint-disable-line @typescript-eslint/no-var-requires
+            ACSChatVersion: require('@microsoft/botframework-webchat-adapter-azure-communication-chat/node_modules/@azure/communication-chat/package.json').version // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
I'm curious why this change was needed; is there a version difference between the direct include and that used in the adapter?